src/Razor/test/testassets/Directory.Build.props

```diff
@@ -26,6 +27,9 @@
 
     <!-- Working around an issue in XDT transforms -->
     <AspNetCoreHostingModel>OutOfProcess</AspNetCoreHostingModel>
+
+    <!-- Turn down the compression level for brotli -->
+    <_BlazorBrotliCompressionLevel>NoCompression</_BlazorBrotliCompressionLevel>
```

Comment:
Why? To make builds faster? Does it make enough of a difference to be worthwhile?

Also, our benchmarking relies on getting realistic brotli-compressed sizes. Just want to check this isn't going to interfere.

Reply:
This is specifically for the integration tests. It takes kinda long: https://github.com/dotnet/aspnetcore/issues/21382